DesignPatternExamples/Assets/Design Patterns/Builder/Scripts/IAirshipBuilder.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace Builder
+{
+    public interface IAirshipBuilder
+    {
+        List<IAirshipModule> airshipModules { get; set; }
+        void SetLeftWing(bool value);
+        void SetRightWing(bool value);
+        void SetLeftWeapon(bool value);
+        void SetRightWeapon(bool value);
+        void SetFrontalWeapon(bool value);
```

Comment:
Following the same principle of `value` parameter name should have context by the method name, consider changing it to the following:

```suggestion
        void SetLeftWingActive(bool value);
        void SetRightWingActive(bool value);
        void SetLeftWeaponActive(bool value);
        void SetRightWeaponActive(bool value);
        void SetFrontalWeaponActive(bool value);
```